karrot/utils/geoip.py

```diff
@@ -24,6 +24,17 @@ def get_client_ip(request):
         return request.META.get('REMOTE_ADDR')
 
 
+@lru_cache()
+def ip_to_city(ip):
```

Comment:
Codecov complains about missing tests for the exception cases. Is that something that should be added?

Reply:
I added some more tests for that now :) ... also better typing, by returning `None` not `False` when it can't find the address.